beacon-chain/db/kv/attestations.go

```diff
@@ -1,89 +1,113 @@
 package kv
```

Comment:
Hey team, please review this file carefully, as most of this logic will be reused for blocks as well.